debugging-via-dotnet-dump/test.sh

```diff
@@ -453,10 +455,10 @@ cat dump.out
 # Skipped on aarch64 due to https://github.com/dotnet/diagnostics/issues/3726
 if [[ "$(uname -m)" != "aarch64" ]]; then
     id=$(grep -F 'System.Threading.Tasks' dump.out | head -1 | cut -d' ' -f 2)
-    dump-analyze "gcroot -all ${id}" > dump.out
```

Comment:
The `-all` option no longer exists in the new versions. It doesn't seem to make a practical difference in terms of things we test, though.

Reply:
`-all` doesn't exist as an option; it's the default in new versions.